libpf/pfelf/file.go

```diff
@@ -489,6 +494,26 @@ func (f *File) GetBuildID() (string, error) {
 	return getBuildIDFromNotes(data)
 }
 
+// GoVersion return the Go version if present and empty string otherwise.
```

Comment:
editorial: perhaps it might be good to document here, or below that `buildinfo.Read` is expensive function in terms of CPU and disk usage as it needs to do linear search of the data from potentially large portion of the large executable